app/services/sign_in/session_revoker.rb

```diff
@@ -61,8 +61,12 @@ def revoking_token
       @revoking_token ||= access_token || refresh_token
     end
 
+    def client_config
+      @client_config ||= SignIn::ClientConfig.find_by(client_id: session.client_id)
```

Comment:
Do we want `client_config` here (and similarly in other classes) to raise an error if not found like in `TokenSerializer`?

Reply:
Oops, I missed this comment for some reason. I think we do want to raise an error when we don't find the client config (if we have many clients it will be a 'much bigger deal' if the client id is unexpected or wrong somehow. For example if someone deletes a profile and expects sessions to stop working after that)

Reply:
I'll add a new PR to make that change